crates/networking/discv5/Cargo.toml

```diff
@@ -10,9 +10,13 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
+alloy-rlp.workspace = true
 anyhow.workspace = true
+ethereum_ssz.workspace = true
+ethereum_ssz_derive.workspace = true
 discv5.workspace = true
 futures.workspace = true
 libp2p.workspace = true
 tokio.workspace = true
 tracing.workspace = true
+ssz_types.workspace = true
```

Comment:
![Uploading image.png…]()

Reply:
this comment is bit unclear as the image is not loading if you were trying to upload something